app/helpers/textual_mixins/textual_paused_state.rb

```diff
@@ -0,0 +1,12 @@
+module TextualMixins::TextualPausedState
+  def textual_paused_state
+    state = @record.enabled?
+    h = {:label => _("Running State"), :value => state ? _("Running") : _("Paused")}
+    if state
```

Comment:
So the current wording is `Running State: Running\Paused`
I feel this is a little misleading to users since its not the external provider thats paused but the data collection from it on miq side.
Do you think it should be `Data Collection Workers: Running\Paused` or something that better indicates that its only on the miq side?
@simon3z @Loicavenel